ci/testgrid/config.yaml

```diff
@@ -49,6 +49,8 @@ default_dashboard_tab:
   num_columns_recent: 10
   code_search_url_template:      # The URL template to visit when searching for changelists
     url: https://github.com/knative/serving/compare/<start-custom-0>...<end-custom-0>
+  alert_options:
+    alert_mail_to_addresses: 'knative-testinfradev@googlegroups.com'
```

Comment:
Created knative-productivity-dev@googlegroups.com to use for alerting purpose.

Reply:
Updated.